h2o-genmodel-extensions/xgboost/src/test/java/hex/genmodel/algos/xgboost/XGBoostJavaMojoModelTest.java

```diff
@@ -16,4 +22,31 @@ public void testObjFunction() { // make sure we have implementation for all supp
     }
   }
 
+  @Test
+  public void testPredictContributionsSerialization() throws Exception {
+    MojoReaderBackend readerBackend = MojoReaderBackendFactory.createReaderBackend(
+            XGBoostJavaMojoModelTest.class.getResource("xgboost_java.zip"),
```

Comment:
I think its pretty straight-forward to generate the MOJO at test-time and I prefer it to commiting the zip into git.